scripts/webpack.js

```diff
@@ -44,8 +44,21 @@ function rules() {
 
 function plugins() {
   return {
-    ignoreEsprima: () => new webpack.IgnorePlugin(/^esprima$/, /js-yaml/),
-    ignoreMomentOptionalDeps: () => new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    ignoreEsprima: () =>
+      new webpack.IgnorePlugin({
+        resourceRegExp: /^esprima$/,
+        contextRegExp: /js-yaml/,
+      }),
+    ignoreMomentOptionalDeps: () =>
+      new webpack.IgnorePlugin({
+        resourceRegExp: /^\.\/locale$/,
+        contextRegExp: /moment$/,
+      }),
+    includePolyfills: () =>
+      new webpack.ProvidePlugin({
+        Buffer: ['buffer', 'Buffer'],
+        process: 'process',
+      }),
```

Comment:
Webpack 5 [no longer includes automatic polyfills for Node.js](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed): 

> In the early days, webpack's aim was to allow running most Node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. Webpack <= 4 ships with polyfills for many of the Node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
>
>While this makes using modules written for Node.js easier, it adds these huge polyfills to the bundle. In many cases these polyfills are unnecessary.
>
>Webpack 5 stops automatically polyfilling these core modules and focus on frontend-compatible modules. Our goal is to improve compatibility with the web platform, where Node.js core modules are not available.
>
>…
>
> It's possible to manually add a polyfill for a Node.js core module. An error message will give a hint on how to achieve that.

This (and the below `fallback` configuration lines) follow those hints.